src-tauri/src/main.rs

```diff
@@ -42,6 +42,8 @@ use mod_management::{
 mod northstar;
 use northstar::get_northstar_version_number;
 
+mod platform_specific;
```

Comment:
you had this before

```
#[cfg(target_os = "windows")]
use platform_specific::windows;


#[cfg(target_os = "linux")]
use platform_specific::linux;
```

Reply:
Nope that just gives an additional error of
```
warning: unused import: `platform_specific::linux`
  --> src/main.rs:49:5
   |
49 | use platform_specific::linux;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default
```
which makes like no sense to me T_T